tests/testthat/examples/depends/orderly.R

```diff
@@ -0,0 +1,2 @@
+orderly3::orderly_depends("explicit", "latest", c(graph.png = "mygraph.png"))
+orderly3::orderly_artefact("Final plot", "graph.png")
```

Comment:
Can we add a test case where the name of the dependency is different from the artefact? Ran into an issue with this in orderly2 which is what https://github.com/vimc/orderly2/pull/8 is addressing. Could also do that in a separate PR.

Reply:
In this example the dependency is being renamed, but there's no "unexpected files" check yet here (it'll be a bit harder to add). Or is it something else that's missing in the orderly2 test?

Reply:
I mean the name of the dependency after being pulled into this report run is different from the name of the artefact. They are the same here as you say. Though actually if we have no unexpected files check here then comment above is not relevant!